File: Education/ListView2/app/src/main/java/com/example/listview2/Main2Activity.java
package com.example.listview2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

public class Main2Activity extends AppCompatActivity {
 TextView txt;
 ImageView img;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        txt=(TextView)findViewById(R.id.View);
        img=(ImageView)findViewById(R.id.imageView);
        img.setImageResource(Integer.parseInt(getIntent().getStringExtra("img")));
       // img.setImageURI();
        txt.setText("Titre "+getIntent().getStringExtra("title")+" \n Déscription :" +getIntent().getStringExtra("disc"));
    }
}

File: MyApplication/FirstAppKotlin/app/src/main/java/com/example/firstappkotlin/Main2Activity.kt
package com.example.firstappkotlin

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.GridLayoutManager
import androidx.recyclerview.widget.RecyclerView

class Main2Activity : AppCompatActivity() {

    @SuppressLint("WrongConstant")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        val recycler: RecyclerView = findViewById(R.id.recycler2)
        recycler.layoutManager = GridLayoutManager(this,2)
        val Colorstab:Array<String> = arrayOf("1","2","3","4","5","6")
        val adapterCustm=AdapterCustm(this,Colorstab)
        recycler.adapter=adapterCustm
    }
}
